.gitignore

```diff
@@ -12,12 +12,6 @@ lcov.info
 .idea
 
 /*.prql
-_*.prql
-**/project/**/*.prql
```

Comment:
These are presumably ignoring committed files?

Reply:
I was testing the cli and had a few directories like this. Now that one of them is commited, we can remove this, yes.